site/content/docs/4.6/components/collapse.md

```diff
@@ -133,7 +133,7 @@ You may use [buttons' sizes utilities]({{< docsref "/components/buttons#sizes" >
       <div class="card">
         <div class="card-header" id="headingOne-3">
           <h2 class="mb-0">
-            <button class="btn btn-link btn-sm" type="button" data-toggle="collapse" data-target="#collapseOne-3" aria-expanded="true" aria-controls="collapseOne-3">
+            <button class="btn btn-link btn-sm text-left" type="button" data-toggle="collapse" data-target="#collapseOne-3" aria-expanded="true" aria-controls="collapseOne-3">
```

Comment:
Could you explain to me please why adding a `.text-left` here? Can't see the difference with or without this class.

Reply:
adding `.text-left` because it's in the spec off accordions and on very very very small screen display is wrong otherwise 
![image](https://user-images.githubusercontent.com/11272238/125246221-ca106580-e2f1-11eb-9217-98412ff06250.png)